cmake/CMakeLists.txt

```diff
@@ -804,7 +804,7 @@ if (onnxruntime_USE_CUDA)
   set(CMAKE_CUDA_STANDARD 11)
   file(TO_CMAKE_PATH ${onnxruntime_CUDNN_HOME} onnxruntime_CUDNN_HOME)
   set(ONNXRUNTIME_CUDA_LIBRARIES ${CUDA_LIBRARIES})
-  list(APPEND ONNXRUNTIME_CUDA_LIBRARIES cublas cudnn curand cufft)
+  list(APPEND ONNXRUNTIME_CUDA_LIBRARIES cublas cudnn curand cufft nvToolsExt)
```

Comment:
Please update setup.py also. Whenever a new cuda lib is added, setup.py must be updated too.

Reply:
Sure. It's done in https://github.com/microsoft/onnxruntime/pull/4013/commits/5e028f5a6892eb9bd36e004706aa5ad6802ec31e's last file